app/javascript/packages/device/index.js

```diff
@@ -5,6 +5,15 @@
  * @return {boolean}
  */
 export function isLikelyMobile() {
+  /**
+   * iPadOS devices no longer list the correct user agent.
+   * As a proxy, we check for the incorrect one (Macintosh)
+   * then test the number of touchpoints, which for iPads will
+   * be 5.
+   */
+  if (/macintosh/i.test(window.navigator.userAgent) && window.navigator.maxTouchPoints === 5) {
+    return true;
+  }
   return /ip(hone|ad|od)|android/i.test(window.navigator.userAgent);
 }
```

Comment:
For these kinds of multi-line comments, can we use `/* ... */` or `//` (my preference) style comments? I would only expect DocBlock style `/** ... */` if it were describing some specific variable or function, rather than how we're using it here to explain the flow of logic.

```suggestion
  // iPadOS devices no longer list the correct user agent. As a proxy, we check for the incorrect
  // one (Macintosh) then test the number of touchpoints, which for iPads will be 5.
  if (/macintosh/i.test(window.navigator.userAgent) && window.navigator.maxTouchPoints === 5) {
    return true;
  }
  return /ip(hone|ad|od)|android/i.test(window.navigator.userAgent);
}
```

Alternatively, we could move this logic into its own function, so that `isLikelyMobile` could have a single return path, and we could include the extra context in the description of the function itself.

```suggestion
  return isIPad() || /iphone|android/i.test(window.navigator.userAgent);
}

/**
 * Returns true if the device is an iPad, or false otherwise.
 * 
 * iPadOS devices no longer list the correct user agent. As a proxy, we check for the incorrect
 * one (Macintosh) then test the number of touchpoints, which for iPads will be 5.
 *
 * @return {boolean}
 */
export function isIPad() {
  const { userAgent, maxTouchPoints } = window.navigator;
  return /ipad/i.test(userAgent) || (/macintosh/i.test(userAgent) && maxTouchPoints === 5);
}
```